<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inicio</class>
 <widget class="QDialog" name="Inicio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sudoku 0.3</string>
  </property>
  <widget class="QPushButton" name="btn_jugar">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Jugar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Salir">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbt_facil">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>43</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Facil</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbt_normal">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>56</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Normal</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbt_azanzado">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Avanzado</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Dificultad">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Dificultad:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbt_experto">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>58</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Experto</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_nombre">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
